### ISSUE TITLE
funny business trigger wheel correlation with cylinder count

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi#3117]
is all this about quick cranking? what's the quick cranking strategy they use?

Nissan 360 CAS number of windows matches cylinder count

v6 nissan is 
![image](https://user-images.githubusercontent.com/48498823/128624750-31b421ad-e533-47cb-b35e-5be19eac4930.png)

i4 nissan is
![image](https://user-images.githubusercontent.com/48498823/128624762-5812248c-a6b4-40ac-a752-9f1ba1fe7d57.png)

**Comment by rusefillc** [comment]
See #3118
See v6 picture for reference https://rusefi.com/forum/viewtopic.php?p=41453#p41453

**Comment by mck1117** [comment]
Yes, nissan used a sort of goofy cam pattern that happens to blip out the next cylinder's number.

Is there new code to write for this?
